palette.mss

```diff
@@ -173,7 +173,7 @@
 @gastronomy-text: darken(@gastronomy-icon, 5%);
 @shop-icon: #666;
 @shop-text: darken(@shop-icon, 5%);
-@amenity-water: #007ae2;
+@amenity-water: #3b029a;
```

Comment:
Is it really better to have this dark purple color instead of grey as `@health-color`?

Reply:
Yes, It would be difficult to see the difference between toilet with or without drinking water.
Plus, colors begin only an issue in dense area like Paris, but in normal city it's ok.